packages/conf-bison/conf-bison.2/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "7895506+MSoegtropIMC@users.noreply.github.com"
+homepage: "https://www.gnu.org/software/bison/"
+bug-reports: "https://lists.gnu.org/mailman/listinfo/bug-bison"
+license: "GPL-3.0-or-later"
+authors: [
+  "Robert Corbett" 
+  "Richard Stallman"
+  "Wilfred Hansen"
+  "Akim Demaille"
+  "Paul Hilfinger" 
+  "Joel E. Denny"
+  "Paolo Bonzini"
+  "Alex Rozenman"
+  "Paul Eggert"
+]
+build: [ "bison" "--help" ]
```

Comment:
This should be more something like:
```
build: [
  ["bison" "--help"] {os-family != "homebrew"}
  ["sh" "-c" "\"$(brew --prefix bison)/bin/bison\""] {os-family = "homebrew"}
]
```
same for flex

Reply:
Indeed, yes. Well spotted!